libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java

```diff
@@ -2,3 +2,65 @@
 // Licensed under the MIT License.
 
 package com.microsoft.bot.ai.qna.models;
+
+/**
+ * Active learning feedback record.
+ */
+public class FeedbackRecod {
+    @JsonProperty("userId")
+    private String userId;
+
+    @JsonProperty("userQuestion")
+    private String userQuestion;
+
+    @JsonProperty("qnaId")
+    private String qnaId;
```

Comment:
This is `Integer`, not `String` (see [here](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.AI.QnA/Models/FeedbackRecord.cs#L32). You should change the getter/setter too.

Be careful with this, you are missing these kind of things.